src/component/Feeds.js

```diff
@@ -1,30 +1,63 @@
-import "../styles/feeds.css";
+import '../styles/feeds.css';
+import { BASEURL } from '../App';
 
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { AiFillDelete } from 'react-icons/ai';
 const Feeds = () => {
+  let pages = 0;
+  const [getPosts, setGetPost] = useState([]);
+  const getBlogs = async () => {
+    try {
+      const { data } = await axios.get(`${BASEURL}?page=${pages}`);
+      const allPost = data?.getBlogPost?.docs;
+      setGetPost((oldPost) => [...oldPost, ...allPost]);
+      pages++;
+    } catch (e) {
+      console.log(e);
+    }
+  };
+  const handleScroll = (e) => {
+    if (
+      window.innerHeight + e.target.documentElement.scrollTop + 1 >=
+      e.target.documentElement.scrollHeight
+    ) {
+      getBlogs();
+    }
+  };
+
+  useEffect(() => {
+    getBlogs();
+    window.addEventListener('scroll', handleScroll);
+  }, []);
```

Comment:
Move `handleScroll` into the `useEffect` and add a cleanup function to the `useEffect`

Reply:
ok sir
how can i stop my useEffect from rerendering sir